### ISSUE TITLE
Continuous aggregate dependency issues on drop schema

### ISSUE THREAD

**Issue by davidkohn88** [timescale/timescaledb#2999]
**Relevant system information:**
TimescaleDB 2.0.0

**Describe the bug**
Dropping a schema and cascading to the continuous aggregate in it doesn't work, error is schema [foo] does not exist

**To Reproduce**

```SQL
-- relevant output in comments
CREATE TABLE foo (ts timestamptz, val float);
SELECT create_hypertable('foo', 'ts');
CREATE SCHEMA test;
CREATE MATERIALIZED VIEW test.foo_agg WITH (timescaledb.continuous) as select time_bucket('5 min', ts), avg(val) FROM foo WITH NO DATA;
-- CREATE MATERIALIZED VIEW 
DROP SCHEMA test CASCADE;
-- NOTICE:  drop cascades to view test.foo_agg
-- ERROR:  schema "test" does not exist
```
**Expected behavior**
I'd expect the schema and the continuous aggregate to be dropped. 

**Actual behavior**
An error that caused neither thing to be dropped, it looks to me like a circular dependency or something like that or some sort of other error in how we're doing dependencies for continuous aggregates.

I tested this both with and without data and it appears to break in both cases.

**Comment by svenklemm** [comment]
This is fixed in 2.1.0

```
ts4=# CREATE SCHEMA test;
test.CREATE SCHEMA
Time: 1.840 ms
ts4=# CREATE MATERIALIZED VIEW test.foo_agg WITH (timescaledb.continuous) as select time_bucket('5 min', ts), avg(val) FROM foo GROUP BY 1 WITH NO DATA;
- CREATE MATERIALIZED VIEW
DROP SCHEMA test CASCADE;
CREATE MATERIALIZED VIEW
Time: 31.594 ms
ts4=# -- CREATE MATERIALIZED VIEW
ts4=# DROP SCHEMA test CASCADE;
NOTICE:  00000: drop cascades to view test.foo_agg
LOCATION:  reportDependentObjects, dependency.c:1217
DROP SCHEMA
Time: 7.948 ms
ts4=# \dx
                                      List of installed extensions
    Name     | Version |   Schema   |                            Description
-------------+---------+------------+-------------------------------------------------------------------
 plpgsql     | 1.0     | pg_catalog | PL/pgSQL procedural language
 timescaledb | 2.1.0   | public     | Enables scalable inserts and complex queries for time-series data
(2 rows)
```

**Comment by davidkohn88** [comment]
Ah, sorry! Thanks!
